### DISCUSSION TITLE
Automatic space after punctuation

### DISCUSSION THREAD

**Discussion by jarikmarwede** [AnySoftKeyboard/AnySoftKeyboard#3664] [Ideas]
This feature would automatically add a space after you type a punctuation character (".", ",", "?"). 
I recently switched to ASK and this is a feature that my previous not open source keyboard had. It did annoy you sometimes when you were typing a URL for example. So it should most likely be turned off by default.
What do you think?

**Comment by nicoursi** [comment]
See #2385 and #2195
